packages/flutter/lib/src/gestures/velocity_tracker.dart

```diff
@@ -180,39 +72,151 @@ class Velocity {
   String toString() => 'Velocity(${pixelsPerSecond.dx.toStringAsFixed(1)}, ${pixelsPerSecond.dy.toStringAsFixed(1)})';
 }
 
-/// Computes a pointer velocity based on data from PointerMove events.
+/// A two dimensional velocity estimate.
+///
+/// VelocityEstimates are computed by [VelocityTracker.getVelocityEstimate]. An
+/// estimate's [confidence] measures how well the the velocity tracker's position
+/// data fit a straight line, [duration] is the time that elapsed between the
+/// first and last position sample used to compute the velocity, and [offset]
+/// is similarly the difference between the first and last positions.
+///
+/// See also:
+///
+/// * VelocityTracker, which computes [VelocityEstimate]s.
+/// * Velocity, which encapsulates (just) a velocity vector and provides some
+///   useful velocity operations.
+class VelocityEstimate {
+  const VelocityEstimate({
```

Comment:
Doc constructor pls.  Usually I just say "Creates a ..." and reiterate the first line of the class docs.